File: Programação Funcional/Básicos/Haskell/03.hs
{- 
	Faça uma função mult5 x que retorne True caso a entrada seja múltiplo de 5 e False caso contrário.
-}
mult5 :: Int -> Bool
mult5 x = x `mod` 5 == 0

-- Função principal para testar
main :: IO ()
main = do
    print (mult5 9)  -- Deve retornar False
    print (mult5 10) -- Deve retornar True

File: Programação Funcional/Básicos/Scala/03.scala
object Main {
  // Função que retorna true se a entrada for múltiplo de 5, false caso contrário
  def mult5(x: Int): Boolean = x % 5 == 0

  def mult5(x: Int): Boolean = {
	x % 5 == 0
  }
	
  // Função principal para testar
  def main(args: Array[String]): Unit = {
    println(mult5(9))  // Deve retornar False
    println(mult5(10)) // Deve retornar True
  }
}